daps-xslt/asciidoc/postprocess.xsl

```diff
@@ -170,4 +172,16 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="d:formalpara[d:para/d:screen]">
+    <xsl:element name="example" namespace="&db5ns;">
+      <xsl:apply-templates select="@*|d:title|d:para/*"/>
+    </xsl:element>
+  </xsl:template>
+
+  <xsl:template match="d:informalexample[d:formalpara[d:para/d:screen]]">
+    <xsl:element name="example" namespace="&db5ns;">
+      <xsl:apply-templates select="@*|d:formalpara/d:title|d:formalpara/d:para/*"/>
```

Comment:
You don't match on the elements that come after the formalpara but are still part of the screen (see my first example in the issue where there is a list coming after the formalpara but still within the example). This means that content would be deleted from the document.

Reply:
Agreed, that was an omission. I'm wondering if we should put the elements that come after `formalpara` _outside_ of the `example`. :thinking:  So I'm thinking about something like this:

```xml
  <xsl:template match="d:informalexample[d:formalpara[d:para/d:screen]]">
    <xsl:element name="example" namespace="&db5ns;">
      <xsl:apply-templates select="@*|d:formalpara/d:title|d:formalpara/d:para/*"/>
    </xsl:element>
    <xsl:apply-templates select="*[not(self::d:formalpara)]"/>
  </xsl:template>
```

Reply:
I think it makes more sense to keep those as part of the example, that is more likely to be the author's intention.

Reply:
My only concern was validation would fail. But let's try it with that approach. See latest commit 579de6a.